### ISSUE TITLE
Cannot read property 'name' of undefined in benchmark-47.js

### ISSUE THREAD

**Issue by borisdiakur** [mathiasbynens/jsperf.com#237]
Here is the jsperf for reproduction of the issue: http://jsperf.com/my-anonymous-vs-named-function

Using Google Chrome Version 44.0.2403.125 (64-bit) on OS X 10.10.4

**Comment by heneryville** [comment]
I'm getting a similar issue.

http://jsperf.com/string-equality-lengths

Google Chrome Version 44.0.2403.130 m on Windows 8.1

**Comment by KyleMit** [comment]
[:+1: ](http://jsperf.com/short-cirucuit)

**Comment by kirbysayshi** [comment]
Yep, also an issue on FF 41.0a2 on Mac 10.10.3 using http://jsperf.com/mout-array-difference-vs-custom

**Comment by gibatronic** [comment]
Also getting a similar issue with this one:
http://jsperf.com/biggest-collatz-conjecture

**Comment by gibatronic** [comment]
(sorry for the reference, it has nothing to do with this thread)

**Comment by lagden** [comment]
Yepp...

My test: http://jsperf.com/anonymous-or-arrow/6

error appears when show the results: 

``` javascript
/* Error via Devtools */
//... 
    function e() {
        var W = ui.browserscope
          , U = []
          , T = frames[P("iframe", W.container)[0].name].document // ✖
          , S = P("text,textpath", T)
          , V = W.uaClass
          , R = false;
//...
```

**Comment by bogdanRada** [comment]
+1

**Comment by mcshaman** [comment]
+1

**Comment by olostan** [comment]
He same error at http://jsperf.com/parse-vs-get/2
However, tests run normally. Just this message after test has been run.

**Comment by prateekjahead** [comment]
Yes getting this error!
http://jsperf.com/caching-object-properties-vs-accessing-them
Also, my code has an object property called 'name'. I was like WTH is wrong with the code! :P

**Comment by martin-sweeny** [comment]
+1

**Comment by toddw** [comment]
+1
